src/OnchainKitProvider.tsx

```diff
@@ -20,17 +21,20 @@ export function OnchainKitProvider({
   if (schemaId && !checkHashLength(schemaId, 64)) {
     throw Error('EAS schemaId must be 64 characters prefixed with "0x"');
   }
+  const walletCapabilities = useCapabilitiesSafe({ chain });
+
   const value = useMemo(() => {
     const onchainKitConfig = {
-      address: address ?? null,
+      address: address ?? null, // this can maybe be updated to account.address
       apiKey: apiKey ?? null,
+      capabilities: walletCapabilities ?? null,
```

Comment:
should this default to every value set as `false`?

the type is `WalletCapabilities` (non-optional parameter)

Reply:
<img width="988" alt="image" src="https://github.com/user-attachments/assets/300329ab-f27e-49e2-929e-cef1727098be">

Reply:
this way we can just use the below code in a component without worrying about null `capabilities`

```
const { capabilities } = useOnchainKit()

if (capabilities.paymaster) {
...
}
```

Reply:
> should this default to every value set as false?

yes